tests/phpunit/tests/modules/wpModules.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @group modules
+ * @covers WP_Modules::get_version_query_string
+ */
+class Tests_WP_Modules extends WP_UnitTestCase {
+	/**
+	 * Tests the functionality of the `get_version_query_string` method to ensure
+	 * proper version strings are returned.
+	 *
+	 * @ticket 56313
+	 *
+	 * @covers WP_Modules::get_version_query_string
```

Comment:
```suggestion
	 * @covers ::get_version_query_string
```

Reply:
Done as part of https://github.com/WordPress/wordpress-develop/pull/5818/commits/1aa45a6198e73869dcf4e642c3d2978888af612d.